Tools/Scripts/libraries/webkitbugspy/webkitbugspy/tests/bugzilla_unittest.py

```diff
@@ -748,3 +748,14 @@ def test_source_changes(self):
             captured.stderr.getvalue(),
             'Bugzilla does not support source changes at this time\n',
         )
+
+    def test_see_also(self):
+        with mocks.Bugzilla(self.URL.split('://')[1], environment=wkmocks.Environment(
+                BUGS_EXAMPLE_COM_USERNAME='tcontributor@example.com',
+                BUGS_EXAMPLE_COM_PASSWORD='password',
+        ), users=mocks.USERS, issues=mocks.ISSUES), mocks.Radar(users=mocks.USERS, issues=mocks.ISSUES, projects=mocks.PROJECTS,):
+
+            tracker = bugzilla.Tracker(self.URL)
+            self.assertEqual(tracker.issue(1).references, [])
+            self.assertTrue(tracker.issue(1).add_related_link('2'))
+            self.assertEqual(tracker.issue(1).references, [tracker.issue('2')])
```

Comment:
I don't understand why this is intermixed with other `references`. Like sure, technically this is the case for the mock given the change in `Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/bugzilla.py`, but I don't see how that would be true for the real Bugzilla?

Reply:
`references` is a property that really only exists in `webkitbugspy` since it refers to any bugs that are "referenced" in the current one. After iterating through the comments, we actually append all issues from the `see_also` field to references. 

I would expect this test to accurately reflect "real" Bugzilla/webkitbugspy behaviour insofar as adding a bug to `see_also` would result it being populated in `references`: See [line 313, bugzilla.py](https://github.com/WebKit/WebKit/blob/016eaf8324d1f1bc84be4aed4d6f052b489ca4c5/Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py#L313).

Reply:
> I would expect this test to accurately reflect "real" Bugzilla/webkitbugspy behaviour insofar as adding a bug to `see_also` would result it being populated in `references`: See [line 313, bugzilla.py](https://github.com/WebKit/WebKit/blob/016eaf8324d1f1bc84be4aed4d6f052b489ca4c5/Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py#L313).

That I was _not_ expecting.

I think I'd prefer we had some separate field, `related_issues` or something?

But maybe I should defer to @JonWBedard there?

Reply:
Based on our discussion, I added a separate field!